src/assets/styles/_form.scss

```diff
@@ -40,8 +41,12 @@ $form-checkbox-icon-size: 12px !default;
 
 $form-select-arrow-color: $form-placeholder-color !default;
 $form-select-arrow-icon-size: 12px !default;
-$forms-select-arrow-right-padding: ($form-select-arrow-icon-size + $form-padding-horizontal + $global-whitespace-small) !default;
-$form-select-arrow-surround-padding: 0 $forms-select-arrow-right-padding 0 $form-padding-horizontal !default;
+$forms-select-arrow-right-padding: (
```

Comment:
Would increasing the line-length to something like `120` give us a bit more room here? Not the end of the world here either, but just comes across a little harder to read.

Reply:
[see what you think of this link](https://prettier.io/docs/en/rationale.html#print-width) ultimately I'll defer to you on what is desirable here.